src/lang/string_view_utf8.hpp

```diff
@@ -71,12 +72,16 @@ class string_view_utf8 {
 
     static uint8_t FILE_getbyte(Attrs &attrs) {
         uint8_t c;
+        if (ftell(attrs.file.f) != attrs.file.startOfs + attrs.file.currentOfs)
```

Comment:
`attrs.file.startOfs + attrs.file.currentOfs` doesn't correspond to the comment at currentOfs
I thought currentOfs will start at startOfs and thus no computation is necessary at this spot.
Considering the semantics of startOfs it would make sense to make currentOfs behave the same (and not being something you have to add to startOfs)